src/Chapter1/EvenFibonacci1.hs

```diff
@@ -0,0 +1,25 @@
+module Chapter1.EvenFibonacci where
```

Comment:
this is not using the approach that is suggested in the second part of the problem. Please reimplement.